src/universal/modules/notifications/containers/Notifications/NotificationsContainer.js

```diff
@@ -5,6 +5,8 @@ import Notifications from 'universal/modules/notifications/components/Notificati
 
 const NotificationsContainer = (props) => {
   const {dispatch, notifications} = props;
+  // to avoid a waterfall we push notifications down the DOMs throat, but here it's worth the wait
```

Comment:
Not really sure what this comment means given the context, but let's keep the metaphors clean 🛁